dyn_em/solve_em.F

```diff
@@ -293,6 +295,13 @@ SUBROUTINE solve_em ( grid , config_flags  &
 ! End of adaptive time step modifications
 !-----------------------------------------------------------------------------
 !
+! Set restart flag value history output time
+!-----------------------------------------------------------------------------
+!
+      restart_flag = .false.
```

Comment:
@negin513 
Negin,
Line up the top restart_flag with the following IF test

Reply:
Thanks for the suggestion. I made that correction.